webpack/webpack.config.dev.js

```diff
@@ -0,0 +1,28 @@
+const path = require("path");
+const webpack = require("webpack");
+const { merge } = require("webpack-merge");
+const common = require("./webpack.config.common");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+var CaseSensitivePathsPlugin = require("case-sensitive-paths-webpack-plugin");
+
+module.exports = merge(common, {
+  mode: "development",
+  devServer: {
+    contentBase: path.join(__dirname, "dev"),
+    port: 3000,
+    publicPath: "http://localhost:3000/dist/",
+    hotOnly: true,
+  },
+  devtool: "inline-source-map",
+  optimization: {
+    minimize: false,
+    splitChunks: {
+      chunks: "all",
+    },
+  },
+  plugins: [
+    new webpack.HotModuleReplacementPlugin(),
+    new HtmlWebpackPlugin(),
```

Comment:
By default `HtmlWebpackPlugin` does not create div with id `root` in autogenerated html file. May be consider to use such option `template: 'public/index.html'` to generate index.html based on your existing html file

Reply:
same for webpack-dev-server too
![image](https://user-images.githubusercontent.com/32191501/89210425-c09e5980-d5c8-11ea-9753-30637c8f3222.png)


after I changed to
```        
    new HtmlWebpackPlugin({
      template: 'public/index.html',
    }),
```
everything start to work

Reply:
Thank, I fixed it locally. But forgot to push)